clients/gtest/csrlsvchol_gtest.cpp

```diff
@@ -66,6 +66,11 @@ template <bool HOST>
 class CSRLSVCHOL_BASE : public ::TestWithParam<csrlsvchol_tuple>
 {
 protected:
+    void SetUp() override
+    {
+        if(hipsolverSpCreate(nullptr) == HIPSOLVER_STATUS_NOT_SUPPORTED)
+            GTEST_SKIP() << "Sparse dependencies could not be loaded";
```

Comment:
This is fine for now. We may at some point want to add a method to require the sparse tests, to ensure that we don't introduce a bug that breaks loading. Maybe we can check if `HIPSOLVER_SPARSE_TESTS_REQUIRED=1` is set in the environment.